packages/core/addon/adapters/spree.js

```diff
@@ -1,15 +1,15 @@
 import Ember from 'ember';
-import DS from 'ember-data';
+import ActiveModelAdapter from 'active-model-adapter';
```

Comment:
@givanse we probably need to do the same thing with the Spree Serializer too yeah?